src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/simplifybooleanreturn/Example1.java

```diff
@@ -0,0 +1,55 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="SimplifyBooleanReturn"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.simplifybooleanreturn;
+
+// xdoc section -- start
+class Example1 {
+
+  boolean cond;
+  int a;
+  int b;
+
+  boolean check1() {
+    if (cond) { // violation, 'Conditional logic can be removed
+      return true;
+    }
+    else {
```

Comment:
please format like 
`} else {`

to make it more compact.
same for others

Reply:
Done